modules/devguide/examples/c/connecting-cert-auth.cc

```diff
@@ -1,53 +1,57 @@
 #include <libcouchbase/couchbase.h>
-#include <string.h>
-#include <stdlib.h>
+#include <libcouchbase/utils.h>
+#include <iostream>
 
 static void
-die(lcb_error_t rc, const char *msg)
-{
-    fprintf(stderr, "%s failed. (0x%x, %s)\n", msg, rc, lcb_strerror(NULL, rc));
+die(lcb_STATUS rc, const char *msg) {
+    std::cerr << "[ERROR] " << msg << ": " << lcb_strerror_short(rc) << std::endl;
     exit(EXIT_FAILURE);
 }
 
 int
-main(int argc, char **argv)
-{
-    lcb_INSTANCE instance;
-    struct lcb_create_st cropts;
-    lcb_error_t rc;
-
-    memset(&cropts, 0, sizeof cropts);
-    cropts.version = 3;
-    cropts.v.v3.connstr = "couchbases://127.0.0.1/default"
-                          "?truststorepath=../etc/x509-cert/SSLCA/clientdir/trust.pem"
-                          "&certpath=../etc/x509-cert/SSLCA/clientdir/client.pem"
-                          "&keypath=../etc/x509-cert/SSLCA/clientdir/client.key";
-
-    rc = lcb_create(&instance, &cropts);
+main(int, char **) {
+    lcb_STATUS rc;
+    std::string username = "some-user";
+    std::string password = "some-password";
+    std::string connection_string =
+            "couchbases://127.0.0.1"
+            "?truststorepath=../etc/x509-cert/SSLCA/clientdir/trust.pem"
```

Comment:
Just wondering, if I were to run this example, would I need to create my own certificate paths? Maybe we can add a short comment to say how we obtain these.

Reply:
Yeah, a comment about which buckets, user/password + RBAC etc. are expected would be really helpful.

Reply:
@maria-robobug and @osfameron - I'll add a reference to the SSL connection documentation. For the SSL examples, the bucket isn't really important. It's just connecting and displaying some proof that it's connected.